index.js

```diff
@@ -14,26 +36,43 @@ function getAuthors( options, callback ) {
 		}
 
 		var tracked = {};
+		options = getOptions( options );
+
 		var authors = result.trimRight().split( "\n" )
-			.concat( (options.priorAuthors || []).reverse() )
-			.reverse()
-			.filter(function( author ) {
-				var first = !tracked[ author ];
-				tracked[ author ] = true;
-				return first;
-			});
+			.concat( (options.priorAuthors || []).reverse() );
+
+		authors = orderBy[ options.order ]( authors );
 
 		callback( null, authors );
 	});
 }
 
+function getOptions( options ) {
+	if ( !orderBy.hasOwnProperty( options.order ) ) {
+		options.order = "date";
+	}
+}
```

Comment:
After this error I decided I should wait until the next day. :)

Done by 5830734